coin/coin_di/src/main/java/com/pavluyk/coin_di/coinDetailedViewModels.kt

```diff
@@ -7,7 +7,7 @@ import org.koin.androidx.viewmodel.dsl.viewModel
 import org.koin.dsl.module
 
 val coinDetailedViewModels = module {
-    viewModel { (param: Bundle?) ->
-        CoinDetailedViewModel(get(), param?.getString(ARG_ID) ?: "")
+    viewModel { (bundle: Bundle?) ->
+        CoinDetailedViewModel(fetchDetailedData = get(), id = bundle?.getString(ARG_ID) ?: "")
```

Comment:
also here for readability order of arguments matters, id generally has top priority, flags after, and some usecases/repos/datastores - lowest